src/components/tooltip/tooltip.js

```diff
@@ -1,38 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Icon, OverlayTrigger, Popover, Tooltip as PFTooltip } from 'patternfly-react';
+import { Icon } from 'patternfly-react';
+import { Popover, Tooltip as PFTooltip } from '@patternfly/react-core';
 import helpers from '../../common/helpers';
 
-const Tooltip = ({ children, tooltip, id, placement, popover, rootClose, trigger, delayShow, ...props }) => {
+const Tooltip = ({ children, id, placement, isPopover, content, rootClose, trigger, delayShow, ...props }) => {
```

Comment:
required:
- remove `rootClose` and `trigger` they were associated with the old "overlay trigger"
   - also remove them from the props, default props 
- remove the spread, `...props`